client/state/selectors/get-email-forwards.js

```diff
@@ -11,6 +11,17 @@ export function getEmailForwards( state, domainName ) {
 	return state?.emailForwarding?.[ domainName ]?.forwards ?? null;
 }
 
+/**
+ * Retrieves the state of the reducer for adding an email forward
```

Comment:
I'm not sure we need to mention the reducer here:
```suggestion
 * Determines whether we are in the process of adding a new email forward to the specified domain.
```